ping.go

```diff
@@ -503,6 +515,24 @@ func (p *Pinger) runLoop(
 	}
 }
 
+func (p *Pinger) CheckInFlightPackets() {
+	// Loop through each item in map
+	if time.Duration(p.TTL) == maxPacketTimeout {
```

Comment:
I think you misunderstand what TTL is. This is the ICMP packet network TTL. AKA, how many router hops. Not time.

Reply:
Not at all; it would be more accurate to say I misunderstand the codebase and changes, and admittedly did not take much time to understand it. 🙃 I saw `TTL` added as a new property since @hamptonmoore's original PR, which had property `PacketTimeout`, and made a fast correlation between the two. I will go back through @hamptonmoore's original PR and re-implement the `PacketTimeout` property as it was before.